lerna-script/test/iterators.spec.js

```diff
@@ -103,6 +104,22 @@ describe('iterators', () => {
           })
       })
     })
+
+    it('should respect concurrency limit', async () => {
```

Comment:
Are you checking here blubird concurrency? 
Maybe better to stub Bluebird and just check that concurrency was passed to it?

Reply:
I'm checking the final goal - max num of concurrent executions. in this context `bluebird` is just an impl detail